internal/mock/storage.go

```diff
@@ -66,6 +66,19 @@ func (m *Storage) TxIterator(
 	panic("not implemented") // TODO: Implement
 }
 
+func (m *Storage) BlockReverseIterator(_, _ uint64) (storage.Iterator[*types.Block], error) {
+	panic("not implemented") // TODO: Implement
+}
+
+func (m *Storage) TxReverseIterator(
+	_,
+	_ uint64,
+	_,
+	_ uint32,
+) (storage.Iterator[*types.TxResult], error) {
+	panic("not implemented") // TODO: Implement
```

Comment:
Leftover?

Reply:
The same as previous comment. I would prefer to test these methods using an integration test instead of implementing a mock iterator.